extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts

```diff
@@ -97,10 +90,67 @@ export class LoginMigrationTargetSelectionPage extends MigrationWizardPage {
 				CSSStyles: { ...styles.BODY_CSS, 'margin': '0' }
 			}).component();
 
+		const preRequisiteListTitle = view.modelBuilder.text()
+			.withProps({
+				value: constants.LOGIN_MIGRATIONS_PRE_REQ_TITLE,
+				CSSStyles: {
+					...styles.LABEL_CSS,
+					'margin': '0px',
+				}
+			}).component();
+
+		const preRequisiteListSubTitle = view.modelBuilder.text()
+			.withProps({
+				value: constants.LOGIN_MIGRATIONS_PRE_REQ_SUBTITLE,
+				CSSStyles: {
+					...styles.BODY_CSS,
+					'margin-top': '10px',
+				}
+			}).component();
+
+		const preRequisiteListElement = view.modelBuilder.text()
+			.withProps({
+				value: [
+					constants.LOGIN_MIGRATIONS_PRE_REQ_1,
+					constants.LOGIN_MIGRATIONS_PRE_REQ_2,
+					constants.LOGIN_MIGRATIONS_PRE_REQ_3,
+					constants.LOGIN_MIGRATIONS_PRE_REQ_4,
+					constants.LOGIN_MIGRATIONS_PRE_REQ_5
+				],
+				CSSStyles: {
+					...styles.BODY_CSS,
+					'padding-left': '20px',
+				}
+			}).component();
+
+		const preRequisiteListInfoBox = this._view.modelBuilder.infoBox()
+			.withProps({
+				style: 'information',
+				text: constants.LOGIN_MIGRATIONS_PRE_REQ_INFO,
+				CSSStyles: { ...styles.BODY_CSS },
+				links: [
+					{ text: constants.VIEW_TUTORIAL, url: 'https://learn.microsoft.com/en-us/azure/dms/tutorial-login-migration-ads' },
```

Comment:
Nit: Define url as a constant

Reply:
This is the pattern followed throughout the code where links are added in infobox, so keeping it the same for consistency.